src/Plugin/Block/PublicationNavigationBlock.php

```diff
@@ -81,6 +93,7 @@ public function build() {
 
     if (!empty($node->book['bid'])) {
       $tree = $this->bookManager->bookTreeAllData($node->book['bid'], $node->book);
+      $this->moduleHandler->alter('localgov_publications_menu_tree', $tree);
```

Comment:
Don't forget to add a theneHandler version here too, as themes may want to also alter the menu.

Reply:
Themehander doesn't have a corresponding alter() method that I can see... I'll check if moduleHander calls the alter hook for themes too.

Reply:
Looks like it's ThemeManager::alter() if you want to let your alter hook be implemented by a theme too. I'll add that.

Reply:
I think I might be confusing that with [ThemeManager::alter](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Theme%21ThemeManager.php/function/ThemeManager%3A%3Aalter/10). I'll do a little more research.